src/content/pages/FileUpload/constants.tsx

```diff
@@ -0,0 +1,19 @@
+﻿export const EXAMPLE = {
+  posterior: {
+    content: {
+      attribute1: [0.15186366972638765, -0.796240857285804],
+      attribute2: [
+        {
+          complex: true,
+          real: 20.274669606394458,
+          imag: -1.9407450062971325
+        },
+        {
+          complex: true,
+          real: 20.36245193237846,
+          imag: -1.486144336535883
+        }
```

Comment:
```suggestion
      attribute2: [
        {
          __complex__: true,
          real: 20.274669606394458,
          imag: -1.9407450062971325
        },
        {
          __complex__: true,
          real: 20.36245193237846,
          imag: -1.486144336535883
        }
```
The expected data format for complex parameters is wrong 
![image](https://user-images.githubusercontent.com/101340211/195008707-7f990564-822e-4e43-a3a2-af6216840f19.png)